jwql/website/apps/jwql/templates/view_exposure.html

```diff
@@ -264,7 +264,7 @@ <h5>Submit Anomaly for Group</h5>
         {% elif 'uncal' in suffixes %}
             <script>change_all_filetypes('uncal', '{{group_root}}', '{{num_ints}}', "{{available_ints}}", "{{total_ints}}", '{{inst}}', "{{ detectors|join(',') }}");</script>
         {% elif suffixes|length == 1 %}
-            <script>change_all_filetypes('{{suffixes.0}}', '{{group_root}}', '{{num_ints}}', "{{available_ints}}", "{{total_ints}}", '{{inst}}, "{{ detectors|join(',') }}";</script>
+            <script>change_all_filetypes('{{suffixes.0}}', '{{group_root}}', '{{num_ints}}', "{{available_ints}}", "{{total_ints}}", '{{inst}}', "{{ detectors|join(',') }}");</script>
```

Comment:
Oh man, the missing `'` is a good catch. Thanks @BradleySappington. Any reason for selecting single quotes over doubles?

Reply:
zero reason, I just fixed the bug but didn't do a lot of code cleanup because I wanted to include it in this release, less changes meant less to test ;)